test/CustomOperatorTest/TestOperator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace CustomOperatorTest\Rx\Operator;
+
+use Rx\ObservableInterface;
+use Rx\ObserverInterface;
+use Rx\Operator\OperatorInterface;
+use Rx\SchedulerInterface;
+
+class TestOperator implements OperatorInterface
+{
+    private $mapTo;
+
+    public function __construct($mapTo)
+    {
+        $this->mapTo = $mapTo;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function __invoke(
+        ObservableInterface $observable,
+        ObserverInterface $observer,
+        SchedulerInterface $scheduler = null
+    ) {
+        return $observable
```

Comment:
I'm leaning toward leaving `ObservableInterface` as it.  Among other things, if you ever wanted to, you could create your own observable that only implements the operators with schedulers that you want.